crates/sui-types/src/event.rs

```diff
@@ -124,35 +127,83 @@ pub enum Event {
         package_id: ObjectID,
     },
     /// Transfer objects to new address / wrap in another object / coin
```

Comment:
nit: comment for CoinBalanceChange instead?

Reply:
good catch!